src/plugins/lua/bindings/lua_ImGuiCustom.cpp

```diff
@@ -23,6 +23,9 @@
 #include <imgui/imgui_internal.h>
 #include <imgui/misc/cpp/imgui_stdlib.h>
 #include <sol/sol.hpp>
+#include <zep/enums.h>
+
+#include "main/ImGuiZepEditor.h"
```

Comment:
We shouldn't be including anything from main in plugins, this is the indicator that the design needs to be changed